search_bar

Aaronkitiibwa opened the pull request:
search bar component will will help our users to filter their content to be able to easily locate information about the desired content

Yesigomwe committed:
Create the login component.( Almost done with the styling)

Yesigomwe committed:
Add config file for mantine

Yesigomwe committed:
Merge pull request #3 from Yesigomwe/create-login-page

Create the login component.

Aaronkitiibwa committed:
adding footer component

Yesigomwe committed:
Landing page

Yesigomwe committed:
Add major components for the application (Landing page,Recipes Categories, Recipe details,Login

Yesigomwe committed:
Reset Home.jsx to remove conflict but all is good

Yesigomwe committed:
Merge branch 'develop' into Footer

Yesigomwe committed:
Merge pull request #6 from Yesigomwe/Footer

Footer

Yesigomwe committed:
Code refactor

Yesigomwe committed:
Add react routing to the application and refactor the code.

Yesigomwe commented:
This was reviewed on the PR with the Footer and got merged. Consider this as a duplication since it already exists.

Yesigomwe reviewed (commented):
Reviewed and it looks fine

Yesigomwe committed:
Code Refactor

Yesigomwe committed:
Create the signup page

geraldokeith committed:
Merge pull request #8 from Yesigomwe/Create_the_signup_page

Create the signup page

Yesigomwe committed:
Merge pull request #7 from Yesigomwe/Create_navigation_to_recipe_details_from_homescreen

Add react routing to the application and refactor the code.

Yesigomwe committed:
Initial works on the About page

Yesigomwe committed:
Create the about page and add data

Yesigomwe committed:
Merge pull request #9 from Yesigomwe/Create_about_page

Create about page

Yesigomwe committed:
Added database

Yesigomwe committed:
Added categories

Yesigomwe committed:
Merge pull request #10 from Yesigomwe/Add_data_to_categories

Add data to categories

Yesigomwe committed:
Add data to the recipes

Yesigomwe committed:
Merge pull request #11 from Yesigomwe/Add_data_to_recipe

Add data to the recipes

Yesigomwe committed:
Add limitation of recipes to 3

Yesigomwe committed:
Add base route to the landing page

Yesigomwe committed:
Add recipe details data

Yesigomwe committed:
Merge pull request #12 from Yesigomwe/Create_recipe_details_model_and_data

Add recipe details data

Yesigomwe committed:
Merge pull request #13 from Yesigomwe/develop

Merge develop to main for production deployment

Yesigomwe committed:
Refactor the api url

Yesigomwe committed:
Delete Backend directory

I am deleting this directory to keep the frontend repo lighter. I have created a different repo for the backend

Yesigomwe committed:
Delete Backend directory

Yesigomwe committed:
Merge pull request #15 from Yesigomwe/main

code refactor

Yesigomwe committed:
Update README.md with application link

Added link to application in read me file

Yesigomwe committed:
Navigation bug fix

Yesigomwe committed:
Merge pull request #14 from Yesigomwe/develop

code refactor

Yesigomwe committed:
Change directory of the images

Yesigomwe committed:
Merge pull request #17 from Yesigomwe/develop

Change directory of the images

Yesigomwe committed:
Merge pull request #18 from Yesigomwe/main

Main

geraldokeith committed:
this is my contribution to the blog and contact pages

Yesigomwe committed:
Merge branch 'develop' into blog-page-content

Yesigomwe committed:
Merge pull request #20 from Yesigomwe/blog-page-content

this is my contribution to the blog and contact pages

Yesigomwe committed:
code refactor

Yesigomwe committed:
code refactor

Yesigomwe committed:
code refactor

Yesigomwe committed:
code refactor
